_develop/connect/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Get Started
+{:.no_toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
```

Comment:
(FYI - for the rest of comments on here, I'll take on a "beginners mindset". assuming the reading is lacking sufficient context)

What's Stacks v2? Do I want to integrate it?